data/competencies.yml

```diff
@@ -953,3 +953,236 @@
   level: senior2
   area: leadership
   domain: null
+
+# Tech lead, Technical
+
+- id: b6787605-ffa6-4243-a29f-2de90eba481a
+  summary: Works with the relevant Principal to develop and maintain a technical vision for the product, project or team
+  examples:
+    - Has meetings with the relevant Principal to plan future milestones
+    - Has a high level technical roadmap (document, slide deck, diagram, etc)
+    - Speaks to the Principal and others outside the team to help develop the technical vision material
+    - Ensures the technical vision material is up to date
+    - Demonstrates that external needs (from internal stakeholders, users, technical trends, historical problems, etc) have influenced the technical vision
+  supportingUrls: []
+  description: null
+  level: tech-lead
+  area: technical
+  domain: null
+
+- id: dfcb7dea-eac6-4aa7-8b8c-09420bfe7f70
+  summary: Breaks down work described in business/user terms into user stories that engineers can work on confidently
+  examples:
+    - Adds technical detail to tickets which are described in business/user terms so that engineers can work on them
+    - Spends time adding technical details/tasks to tickets or related background documents
+    - Engineers routinely pick up a variety of tickets
+    - Engineers can read a ticket and generally estimate consistently
+    - Can go on holiday and have the team continue delivering smoothly
+  supportingUrls: []
+  description: null
+  level: tech-lead
+  area: technical
+  domain: null
+
+- id: 3f5ec39a-9d23-4f50-bd7d-99746a4b709b
+  summary: Ensures runbooks are up to date and correct
```

Comment:
[suggestion] Runbooks are great, but this feels like an example of `Ensures the technology’s documentation is at an appropriate level and is kept up to date` rather than a standalone competency

Reply:
Thanks - yes, that was far too specific. The intent was to help tech leads think about not just the build but also the supportability of their system. Therefore I've updated the summary to (hopefully) convey that better, and also adjusted a couple of the examples to make them slightly less dependent on runbooks. I hope that looks better now.